### DISCUSSION TITLE
Convert CMYK tif without clipping path to a transparent PNG in RGB.

### DISCUSSION THREAD

**Discussion by rarreghini** [ImageMagick/ImageMagick#3382] [Digital Image Processing] [answered]
Hi all,

I am not able to convert a CMYK tif to a transparent PNG in RGB.
The source image does not have any clipping path information or any mask.
The transparency is implemented AFAIK as the A channel in the CMYKA

The source image is in 
https://www.dropbox.com/s/xu2ls4akubfuuex/36525_sekgel_1%283x%29_bc_ES_GHS_pack.tif?dl=0

I tried a simple conversion with profile specification
convert -density 300 source.tif[0] -profile "Coated_Fogra39L_VIGC_300.icc" -profile "sRGB-IEC61966-2.1.icc" result.png

But the result has white background
result file is in:
https://www.dropbox.com/s/e3y45tn9p6acsxx/36525_sekgel_1%283x%29_bc_ES_GHS_pack.tif-cmyk.png?dl=0

Tried setting -alpha transparent but result is a totally transparent image

Any suggestion or help will be really appreciated.
 
Thank you

**Comment by snibgo** [comment, marked as answer]
What version of IM, on what platform? If IM v7, I suggest you use "magick" instead of "convert".

"identfy -verbose" tells us your file has two images, both with CMYK ICC profiles. The first image [0] has no alpha channel. The second image [1] does have alpha, so I suppose that is the image you want.

As it has an embedded profile, your first "-profile" is superfluous.
```
magick "36525_sekgel_1(3x)_bc_ES_GHS_pack.tif"[1] -profile "sRGB-IEC61966-2.1.icc" out.png
```

**Reply by rarreghini** [reply]
that did the trick! tks a lot.
Did not see the difference in 2 images in the identfy -verbose result. Once you found it it's pretty obvious.

BTW I'm using ImageMagick 6.9.10-96 Q16 x64 2020-02-24  on windows.
